max number of policies (#2518)
elastic/observability-docs: docs/en/ingest-management/agent-policies.asciidoc

that you added. Lack of connectivity will prevent the {agent}
from checking in with the {fleet-server} and receiving policy updates, but the agents
will still forward data to the cluster.

[discrete]
[[agent-policy-scale]]

> == Scale of Policies Supported

Reviewer: I have a few thoughts about this section:

* "Scale of policies supported" doesn't seem intuitive unless you think users will search on this phrase. Maybe replace with "Policy limits" or "Policy scaling recommendations"?
* I think it would be better to have this section appear earlier in the topic, maybe under the section about policy benefits?

Author: agree on both points yes. it is a recommendation indeed.

> == Scale of Policies Supported

Reviewer: ```suggestion
== Policy scaling recommendations
```

Author: sounds good.

> Maximum number of Elastic Agent policies supported by {fleet} is 500. You may experience slowness in response when a higher number of policies are configured. 

Reviewer: ```suggestion
The maximum number of {agent} policies supported by {fleet} is 500. If more than 500 policies are configured, {fleet} UI performance might be affected.
```
"You may experience slowness" is grammatically vague (the user does not experience slowness).

> Maximum number of Elastic Agent policies supported in a single {fleet} is 500. You may experience slowness in response when a higher number of policies are configured. 

Reviewer: I have a couple questions before I can edit this part:

* What do we mean by a "single Fleet"? Are we talking about one instance of Kibana with Fleet? My concern here is that users will conflate a "single Fleet" with the number of Fleet Servers they have deployed. Can you think of a better way to describe this?
* How does the user experience the slow response that you describe? Are you talking about UI performance? Or is there some delay in the data being collected, parsed, and indexed into Elasticsearch?

Author: @dedemorton I modified it to just say "fleet". Slowness in response is in UI. If you feel that is too much information we can drop it. thanks.
